docs/src/intro-python.md

```diff
@@ -15,6 +15,15 @@ pip install playwright
 playwright install
 ```
 
+Use conda to install Playwright in your Python project. See [system requirements](#system-requirements).
```

Comment:
I think it would be nicer if we would add something like that:

```
See [system requirements](#system-requirements).

### PIP

pip install playwright
playwright install

### Conda

conda config --add channels conda-forge
conda config --add channels microsoft
conda install playwright
playwright install

```

also please use `bash` instead of `sh`.